source/refills/tooltip.html.erb

```diff
@@ -0,0 +1,3 @@
+<a href="#0" class="tooltipped" aria-label="Tooltip description">
```

Comment:
Having a separate element for the tooltip and using `aria-describedby` might give us the results we are looking for.